apps/pollbook/frontend/src/unconfigured_screen.tsx

```diff
@@ -124,6 +123,20 @@ export function UnconfiguredSystemAdminScreen(): JSX.Element {
     (p) => p.electionId && p.status === PollbookConnectionStatus.WrongElection
   );
   if (!isOnline || configuredPollbooks.length <= 0) {
+    if (hadConfigurationError) {
+      return (
+        <FullScreenMessage
+          title="Failed to configure VxPollBook"
```

Comment:
probably for another, lower-pri PR but I think we use both `VxPollBook` and `VxPollbook` in the product. IIRC we landed on `VxPollBook` so we should standardize at some point

Reply:
Agree it should be `VxPollBook`, but will keep an eye out for that in the copy pass at which point we will standardize.